index.html

```diff
@@ -129,7 +129,61 @@ <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
         'contract'
       ],
       searchFullText: true,
-      source: data
+      source: data,
+      templates: {
+        message: function(type, message) {
+          html = '<p class="searchError">Aucune API ne correspond à votre recherche</p>'
+          + '<a href="' + action.url + '" class="action">'
+          +   action.text
+          + '</a>';
+          return html;
+        },
+        standard: function(response, fields) {
+          var
+            html = ''
```

Comment:
Don't ever get into the habit of writing JS code like that. You will [trigger ASI in ways you probably don't want to care about](http://www.2ality.com/2011/05/semicolon-insertion.html).